apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.GenerateOverloads/Methods/ClearDatasetLabels.xml

```diff
@@ -0,0 +1,13 @@
+﻿<Method Name="ClearDatasetLabels"
+        TargetType="Dataset"
+        ReturnType="IDictionary&lt;string, string&gt;">
```

Comment:
Only just seen this one. We can change that later if we want... it's *relatively* rare, and this is only for BigQuery. Going for cheap and cheerful where possible :)